packages/fxa-payments-server/src/lib/stripe.ts

```diff
@@ -167,6 +172,7 @@ export async function handleSubscriptionPayment({
         productId: selectedPlan.product_id,
         paymentMethodId: paymentMethod.id,
         idempotencyKey,
+        promotionCode: promotionCode || undefined,
```

Comment:
nit: ` || undefined` is unnecessary here also.

Reply:
removed